language.go

```diff
@@ -65,35 +65,33 @@ func (language *GoLanguageModel) Prepare(model *surface.Model) {
 		m.HandlerName = "Handle" + m.Name
 		m.ProcessorName = m.Name
 		m.ClientName = m.Name
+		m.ResponsesTypeName = strings.Title(filteredTypeName(m.ResponsesTypeName))
 	}
+	gnostic_grpc.AdjustSurfaceModel(model, inputDocumentType)
 }
 
-func goParameterName(name string) string {
+func goParameterName(originalName string, t string) string {
+	name := gnostic_grpc.CleanName(originalName)
+	if len(name) == 0 {
```

Comment:
Can we do string comparison here instead of len check? `if name == "" {`

Reply:
Done